src/c4/test/tstifpstream.cc

```diff
@@ -0,0 +1,161 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   c4/test/tstifpstream.cc
+ * \author Mathew Cleveland
+ * \date   Wed Apr 28 09:31:51 2010
```

Comment:
Hehe, a more accurate date may be in order, unless you've really had this sitting around since 2010 :)

Reply:
"We will release no code before its time."

Reply:
Fixed.